ilert/README.md

```diff
@@ -0,0 +1,88 @@
+## Overview
+
+Use the Datadog-iLert integration to send Datadog alerts to iLert and seamlessly take actions on these alerts within the iLert platform.
+
+Integrate with iLert to:
+
+- Trigger and resolve incidents from Datadog
+- Tackle incidents and set up escalation policies as they occur
+- Set up a daily reminder of who is on-call
+
+## Setup
+
+### In iLert
+
+#### Create Datadog alert source
+
+1. Switch to the **Alert Sources** tab and click on the "Create new alert source" button
+
+2. Assign name and select escalation chain
+
+3. Select **Datadog** in the Integration type field and save.
+
+   ![iLert Alert Source New][1]
+
+4. On the next page a **Webhook URL** is generated. You will need this URL at the bottom of the setup in Datadog.
+
+   ![iLert Alert Source View][2]
+
+### In Datadog
+
+#### Add iLert Webhook as alerting channel
+
+1. Go to Datadog integrations page and **install Webhooks integration**: https://app.datadoghq.com/account/settings#integrations
```

Comment:
```suggestion
1. Go to Datadog integrations page and [**install Webhooks integration**][7]: 
```
Let's add this link ref: https://app.datadoghq.com/account/settings#integrations